tests/unit/services/cookies-test.js

```diff
@@ -235,18 +237,29 @@ describeModule('service:cookies', 'CookiesService', {}, function() {
 
   describe('in the FastBoot server', function() {
     beforeEach(function() {
+
+      let request = EmberOject.extend({
+        init() {
+          this._super(...arguments);
+          this.cookies = {};
+          this.protocol;
+          this._host;
```

Comment:
Yes, you are correct. I just thought to add there to make it more clear what are the options we are using. Any how, removed.